src/components/Dropdown/FilterDropdown/FilterDropdown.tsx

```diff
@@ -139,10 +144,17 @@ export const FilterDropdown: FC<Props & ElementProps> = ({
   )
 }
 
-const IsFilteredIconWrapper = styled.span<{ isFiltered: boolean; themes: Theme }>`
+const IsFilteredIconWrapper = styled.span<{
+  isFiltered: boolean
+  themes: Theme
+  size?: ButtonProps['size']
+}>`
   position: relative;
-  color: ${({ isFiltered, themes }) => isFiltered ? themes.color.MAIN : themes.color.TEXT_BLACK};
+  color: ${({ isFiltered, themes }) => (isFiltered ? themes.color.MAIN : themes.color.TEXT_BLACK)};
   line-height: 1;
+  ${({ themes: { space }, size }) => css`
+    transform: translate(${size === 's' ? space(-0.25) : 0});
+  `}
```

Comment:
意図を明確にし、不要なスタイルを当てないための提案です!
（実際に動くかどうかは確認してません:pray:）

また、translate は2軸を指定する関数で、1値しか設定しない場合にY軸が0で初期化されてしまうため translateX の使用を勧めます!

```suggestion
  ${size === 's' && `
    ${({ themes: { space } }) => css`
      transform: translateX(${space(-0.25)});
    `}
  `}
```

Reply:
消したかったので提案いただき助かりました！！（styled-component理解不足ですみません）